### PR TITLE
Chi 1118 date filters

### PR DESCRIPTION
**Pull request by stephenhand**
TODO: Some renames & comments, tests, portuguese translations

Primary reviewer: @murilovmachado 

## Description

* Adds date filters to the cases list
* Gives multi-select filters the same white 'active' styling as date filters when they are applied

### Checklist
- [ X ] Corresponding issue has been opened
- [ NO ] New tests added
- [ NO ] Strings are localized
- [ X ] Tested for chat contacts
- [ NO ] Tested for call contacts

### Related Issues
CHI-1118

### Verification steps

Test new functionality vs mockups & documented requirements

### CODE REVIEW

**Comment by murilovmachado** [comment]
Looking great! I',m still reviewing it, but there're some small things from the latest mockups that are not 100% matched yet:
- The label should be `Date Fiters` instead of `Date Range`
- There should be an icon before `Date Filters`
- There's no box-shadow (or border) around the date filters 
- On Custom Date Range, there should be the labels `Start Date` and `End Date` above the date inputs
- The divisor line looks more lighter/softer on the mockups

Mockups:
image

This PR:
image

Other than that, things to notice:
- The filters should be closable on hitting ESC key
- Once the filter is open, navigating through the keyboard should not leave the dialog, unless we close it. (That's the behavior we have for all our dialogs)
- On the Mockups/Accessibility discussions, we've agreed that the clear and apply buttons would always be active (originally Jana had designed similar to this PR, but the accessibility people advised us to make them all active for the filters)

Here are the [Mockups, for reference.
](https://marvelapp.com/prototype/ai502c9/screen/86188499/handoff)

File: plugin-hrm-form/src/HrmFormPlugin.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ let sharedStateClient: SyncClient;
 export const getConfig = () => {
   const manager = Flex.Manager.getInstance();
 
-  const hrmBaseUrl = `${manager.serviceConfiguration.attributes.hrm_base_url}/${manager.serviceConfiguration.attributes.hrm_api_version}/accounts/${manager.workerClient.accountSid}`;
+  const hrmBaseUrl = `http://localhost:3000/${manager.serviceConfiguration.attributes.hrm_api_version}/accounts/${manager.workerClient.accountSid}`;
```
Comment (murilovmachado) [inline plugin-hrm-form/src/HrmFormPlugin.tsx:26]: Remember to undo this piece that points to localhost

File: plugin-hrm-form/src/components/caseList/CaseListTable.tsx [hunk 2]
```diff
@@ -45,6 +46,7 @@ const CaseListTable: React.FC<Props> = ({
   counselorsHash,
   currentDefinitionVersion,
 }) => {
+  const { featureFlags } = getConfig();
```
Comment (murilovmachado) [inline plugin-hrm-form/src/components/caseList/CaseListTable.tsx:49]: Is is being used in this file?

File: plugin-hrm-form/src/translations/pt-BR/flexUI.json [hunk 3]
```diff
@@ -284,6 +284,19 @@
   "CaseList-Filters-Apply": "Aplicar",
   "CaseList-Filters-Clear": "Limpar",
   "CaseList-SomethingWentWrong": "Algo deu errado enquanto carregava a lista de casos. Por favor, tente novamente em um momento.",
+  "CaseList-Filters-DialogTitlePrefix": "Filtrar por:",
+  "CaseList-Filters-DateFiltersLabel": "Período",
+  "CaseList-Filters-DateFilter-CreatedAt": "Aberto",
+  "CaseList-Filters-DateFilter-UpdatedAt": "Atualizada",
+  "CaseList-Filters-DateFilter-FollowUpDate": "Acompanhamento",
+  "CaseList-Filters-DateFilter-CustomRange": "para",
```
Comment (murilovmachado) [inline plugin-hrm-form/src/translations/pt-BR/flexUI.json:292]: I'd change `CaseList-Filters-DateFilter-CustomRange` value from `para` to `até`, it's more used in this scenario.

**Review by murilovmachado — APPROVED** [review]
Code looks great!

File: plugin-hrm-form/src/components/caseList/CaseListTable.tsx (hunk 2 above)
Comment (stephenhand) [reply plugin-hrm-form/src/components/caseList/CaseListTable.tsx]: No, deleted

**Pull request merged**